chrome/content/quicktext.js

```diff
@@ -801,16 +802,4 @@ var quicktext = {
         break;
     }
   }
-,
-  createInstance: function(aOuter, aIID)
-  {
-    if (aOuter != null) throw Components.results.NS_ERROR_NO_AGGREGATION;
-    return policy;
-  }
-,
-  QueryInterface: XPCOMUtils.generateQI([
-    Components.interfaces.nsIObserver,
```

Comment:
XPCOM objects passed to xpcom functions will need to keep a QueryInterface function, otherwise that function may have difficulty knowing what shape the object has. You likely just need to keep nsIObserver

Reply:
I think I removed all usage of XPCOM and the quicktext object in quicktext.js is not passed into anywhere, except here:
https://github.com/thundernest/quicktext/blob/pro/chrome/content/quicktext.js#L35

but that observer function is self-made and just adds it to an array. No special stuff.

Do you see any place where it is passed to an xpcom function? Same in settings.js

Reply:
Nope, just looked at the diff. Totally fine if it is not passed to something else.